clients/privacy-center/packages/fides-consent/src/lib/cookie.ts

```diff
@@ -107,3 +112,29 @@ export const setConsentCookie = (cookieKeyConsent: CookieKeyConsent) => {
     CODEC
   );
 };
+
+export const setConsentCookieAcceptAll = (defaults: CookieKeyConsent): void => {
```

Comment:
Review note: requiring the `defaults` here adds some messiness to the `fides-consent.ts` file and `consent-banner.ts` as well - we end up passing this `defaults` object around files to get it back here. Feels a bit error-prone.

Reply:
My plan for this was to leave `cookie.ts` as pure functions and keep all `Fides` aware state inside of `index.ts` (or split out that stateful code to its own file if its growing). So in that situation the banner would just interact with a stateful function that would know about the defaults/configuration.

Reply:
Presumably you mean `fides-consent.ts` not `index.ts` right? The latter is just a list of `export * from..`

If so, I'm following. Your instinct here would be to do something like this...
```typescript
// fides-consent.ts
const setConsentCookieAcceptAll = () => {
  ...do stuff with the locally scoped state vars like `consent`, `defaults`, etc.
};

const Fides = {
  consent,

  // Defining this inline for clarity
  banner: async (extraOptions) => {
    // Initialize the banner, providing callbacks to functions here that need state
    initBanner({
      acceptAllCallback: setConsentCookieAcceptAll,
      rejectAllCallback: setConsentCookieRejectAll,
      ...extraOptions
    });
  },
  getBannerOptions,
  gtm,
  meta,
  shopify,
};
```